airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -83,6 +100,16 @@ class Config:
     resourceRequirements: ResourceRequirements
 
 
+class ConnectorBreakingChanges(BaseModel):
+    class Config:
+        extra = Extra.allow
+
+    __root__: Dict[constr(regex=r"^\d+\.\d+\.\d+$"), VersionBreakingChange] = Field(
+        ...,
+        description="Each entry denotes a breaking change in a specific version of a connector that requires user action to upgrade.",
+    )
```

Comment:
I am not sure it matters, but I know that this was originally designed as a list instead of a dict. I think a dict makes sense as we shouldn't expect multiple entries for one key, and the ordering is going to be done based on semver and not necessarily the order that the entries came in - but wanted to call it out just in case

Reply:
```
   additionalProperties: true
    minProperties: 1
    patternProperties:
      "^\\d+\\.\\d+\\.\\d+$":
        $ref: "#/definitions/VersionBreakingChange"
```
the `minProperties: 1` and `additionalProperties: true` sort of indicates to me that a list might be better here? idk

Reply:
ahh sorry, this was an atrifact from originally going the list route. Ill set this to false and add a test case to prove no additional props are allowed